nixnet/types.py

```diff
@@ -6,6 +6,8 @@
 import collections
 import typing  # NOQA: F401
 
+# flake8: NOQA: E501
+
```

Comment:
We should isolate suppressions to the lines they impact
1. So we know if they can be removed
2. So we still get reports on other lines